src/sql/plot.py

```diff
@@ -170,26 +180,88 @@ def _compute_conf_interval(N, med, iqr):
 
     # compute a single array of outliers
     stats["fliers"] = np.array(
-        _between(con, table, column, stats["whislo"], stats["whishi"])
+        _between(con, table, column, stats["whislo"], stats["whishi"], with_=with_)
     )
 
     # add in the remaining stats
     stats["q1"], stats["med"], stats["q3"] = q1, med, q3
 
-    # output is a list of dicts
-    bxpstats = [{k: v for k, v in stats.items()}]
+    bxpstats = {k: v for k, v in stats.items()}
```

Comment:
What happened to the comment here?

Reply:
I deleted it because it's no longer relevant, the function now returns a dictionary, not a list of dictionaries